release-calendar/src/client/models/release-event.ts

```diff
@@ -14,35 +14,29 @@
  * limitations under the License.
  */
 
-import {Channel} from '../../types';
-import {EventSourceInput} from '@fullcalendar/core/structs/event-source';
+import {Channel, Promotion} from '../../types';
 import {ReleaseEventInput} from './view-models';
 
-export function getAllPromotionEvents(
+export function getAllEvents(
   events: ReleaseEventInput[],
-): Map<Channel, EventSourceInput> {
+): Map<Channel, ReleaseEventInput[]> {
   const eventInputs = new Map<Channel, ReleaseEventInput[]>();
   events.forEach((event) => {
     const channelEvents = eventInputs.get(event.channel) || [];
     eventInputs.set(event.channel, [...channelEvents, event]);
   });
+  console.log(eventInputs);
```

Comment:
Debugging leftover?

Reply:
Drats, a mistake I make until the very end. :(

Reply:
Haha don't worry, literally everyone does that all the time no matter how much experience we have